### PR TITLE
FailoverClustering Bulk Fixup (3/4) - Line formatting updates

### PR DESCRIPTION
**Pull request by robinharwood**
Bulk reflowed text and header formatting update before applying further quality updates.

This will be followed by separate PRs for 
	1. Text formatting updates
		a. Bulk formatting checklist update before applying further quality updates:
			- cmdlet name - Change bolds to code spans
			- cmdlet/vars - Change bar ecmdlet/var to code spans
			- parameters - change italic formatting to bold
			- parameters - change code span formatting to bold
			- change italics to use underscores
			- change bold-underscores to asterisk
			- change bold keystrokes to kbd
			- Use contractions
			- Fix indented code blocks
			- Find bad syntax reflows
	2. Update example formatting
		a. Bulk updating example formatting before content updates

### CODE REVIEW

**Comment by robinharwood** [comment]
Hi @michaeltlombardi / @sdwheeler,

I believe this PR is release ready if you would be kind enough to review and if appropriate sign-off/merge 😊

The main update was to reflow and fix any line return issues.

Please let me know if you have any questions or if I need to apply any other updates.

Thanks Robin

File: docset/winserver2022-ps/failoverclusters/Test-Cluster.md [hunk 1]
```diff
@@ -207,22 +219,22 @@ Accept wildcard characters: False
...
+offline or failed state. The acceptable values for this parameter are:
+
+- Int32, Int64, Uint32, Uint64. A number that represents a master boot record (MBR) signature of the
+  disk.
+- **System.String**. A string that represents a master boot record (MBR) signature of the disk,
+  hexadecimal format is supported.
+- **System.String**. A string that represents the GUID of a GPT disk.
+- **ClusterResource**. A cluster resource object that represents a clustered disk.
+- **CimInstance#MSFT_Disk**. An object returned from Get-Disk, from the Windows PowerShell® storage
+  module.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Test-Cluster.md:237]:
```suggestion
- Int32, Int64, Uint32, Uint64. A number that represents a master boot record (MBR) signature of the
  disk.

- **System.String**. A string that represents a master boot record (MBR) signature of the disk,
  hexadecimal format is supported.

- **System.String**. A string that represents the GUID of a GPT disk.

- **ClusterResource**. A cluster resource object that represents a clustered disk.

- **CimInstance#MSFT_Disk**. An object returned from Get-Disk, from the Windows PowerShell® storage
  module.
```

File: docset/winserver2022-ps/failoverclusters/Test-Cluster.md [hunk 2]
```diff
@@ -330,15 +348,19 @@ Accept wildcard characters: False
...
+Cluster Shared Volume, you must also specify the **Force** parameter to take the storage pool
+offline for the duration of the storage tests. Otherwise, the specified storage pool must be taken
+offline before the storage tests. If the **Pool** parameter is not specified, storage tests run on
+all storage pools that are available for use in the cluster or that are in the cluster resource
+offline or failed state. The acceptable values for this parameter are:
+
+- **System.String**: A string that represents the name of the clustered storage pool or pools.
+- **ClusterResource**: A cluster resource object that represents a clustered storage pool.
+- **CimInstance#MSFT_StoragePool**: An object returned from **Get-StoragePool**, from the Windows
+  PowerShell storage module.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Test-Cluster.md:363]:
```suggestion
- **System.String**: A string that represents the name of the clustered storage pool or pools.

- **ClusterResource**: A cluster resource object that represents a clustered storage pool.

- **CimInstance#MSFT_StoragePool**: An object returned from **Get-StoragePool**, from the Windows
  PowerShell storage module.
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
Minor notes for syntax that breaks in-console help

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**